src/prefabs/updateForm.tsx

```diff
@@ -675,6 +683,9 @@ const beforeCreate = ({
             case 'thisPage':
               saveThisPage();
               break;
+            case 'other':
+              save({ ...originalPrefab });
```

Comment:
I'm pretty sure this can just be `save(originalPrefab)`

Reply:
Ah yeah I see, in that case we can skip it since it also has nothing to do with this PR